apps/webClient/src/components/View/index.tsx

```diff
@@ -85,14 +85,21 @@ const View: React.FC<Props> = ({ children }) => {
     return clsx(coloredPageRoot);
   };
 
+  const logoBoxMarginTop = useCallback(() => (isSmallScreen ? 2 : 16), [
+    isSmallScreen,
+  ]);
+  const logoHeight = useCallback(() => (isSmallScreen ? 60 : 80), [
+    isSmallScreen,
+  ]);
+
   return (
     <div className={derivePageRootBackgroundColor(pathname)}>
       <Container maxWidth="md" className={classes.main}>
         {isSmallScreen && <MobileNav />}
-        <Box mt={16} mb={12}>
+        <Box mt={logoBoxMarginTop()} mb={12}>
```

Comment:
very clear name, which is awesome.  Function names should always be verbs: `deriveLogoBoxMarginTop` or `get...` are generally used but I'm open to other words